.circleci/config.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           name: gradle build
           command: |
             cd ~/repo/examples/gradle-example
-            ./gradlew build -x test
+            ./gradlew build -x test -Ptmc-auth.version=LOCAL-SNAPSHOT
```

Comment:
Curious about changes in this file. We are now running example tests using a local snapshot of `tmc-auth` whereas earlier it was using a released version. Was that a bug? Should it have been using local snapshot all this time?

Reply:
Before this change when running examples in the pipeline they were using a released version.
```
ext {
    tmcAuthVersion = project.hasProperty('tmc-auth.version') ? project.getProperty('tmc-auth' +
            '.version') : '3.1.1-beta'
}
```
Because our changes are in LOCAL-SNAPSHOT the example build was failing searching for the sdk-common dependency transitively via `tmc-auth`.
With this change we are ensuring that each example uses the latest `tmc-auth` when building and testing in the pipeline.